src/Ombi/ClientApp/app/requests/tvrequests.component.ts

```diff
@@ -227,15 +226,7 @@ export class TvRequestsComponent implements OnInit {
             val.data.posterPath = "../../../images/default_tv_poster.png";
         }
 
-        if (val.data.imdbId.length === 0) {
-            this.searchService.getShowInformationTreeNode(val.data.tvDbId).subscribe(x => {
-                    if (x.data) {
-                        val.data.imdbId = "https://www.tvmaze.com/shows/" + x.data.seriesId;
-                    }
-            });   
-        } else {
-            val.data.imdbId = "http://www.imdb.com/title/" + val.data.imdbId + "/";
-        }
+        val.data.imdbId = "http://www.imdb.com/title/" + val.data.imdbId + "/";
```

Comment:
This is not really the `imdbId`. I assume we use this on the UI. I'd prefer if we just do this in the UI